File: app/src/main/java/mchehab/com/twowaydatabinding/Converter.java
package mchehab.com.twowaydatabinding;

import android.databinding.InverseMethod;

/**
 * Created by muhammadchehab on 1/5/18.
 */

public class Converter {

    @InverseMethod("toInt")
    public static String toString(int value) {
        return "" + value;
    }

    public static int toInt(String value) {
        return Integer.parseInt(value);
    }
}

File: kotlinapp/src/main/java/mchehab/com/kotlinapp/Converter.kt
package mchehab.com.kotlinapp

import android.databinding.InverseMethod

/**
 * Created by muhammadchehab on 1/5/18.
 */
class Converter{
    @InverseMethod("toInt")
    fun toString(value: Int): String {
        return "" + value
    }

    fun toInt(value: String): Int {
        return Integer.parseInt(value)
    }
}